Gemfile-rails-5.0.x

```diff
@@ -0,0 +1,29 @@
+source "http://rubygems.org"
+
+# Declare your gem's dependencies in rules.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gemspec
+
+# jquery-rails is used by the dummy application
+gem "activeadmin", '1.0.0'
```

Comment:
nit: what about putting all duplicated gems from all Gemfiles to .gemspec? Should be easier to maintain them in the future